docs/running_psalm/issues/AmbiguousConstantInheritance.md

```diff
@@ -0,0 +1,41 @@
+# AmbiguousConstantInheritance
+
+Emitted when a constant is inherited from multiple sources.
+
+```php
+<?php
+
+interface Foo
+{
+    /** @var non-empty-string */
+    public const CONSTANT='foo';
+}
+
+interface Bar
+{
+    /**
+     * @var non-empty-string
+     */
+    public const CONSTANT='bar';
+}
+
+interface Baz extends Foo, Bar {}
```

Comment:
Is this an issue when the type are compatibles?

Reply:
It's an issue even when it's the same exact thing: https://3v4l.org/tjYML

Reply:
oh, I thought "ambiguous" was too tame a word for something that broke at compile time, I was wrong!